src/manifolds/SymmetricPositiveDefiniteGeneralizedBuresWasserstein.jl

```diff
@@ -0,0 +1,146 @@
+@doc raw"""
+    GeneralizedBurresWassertseinMetric{T<:AbstractMatrix} <: AbstractMetric
+
+The generalized Bures Wasserstein metric for symmetric positive definite matrices, see[^HanMishraJawanpuriaGao2021].
+
+This metric internally stores the symmetric positive definite matrix ``M`` to generalise the metric,
+where the name also follows the mentioned preprint.
+
+[^^HanMishraJawanpuriaGao2021]:
+    > Han, A., Mishra, B., Jawanpuria, P., Gao, J.:
+    > Generalized Bures-Wasserstein geometry for positive definite matrices
+    > arXiv: [2110.10464](https://arxiv.org/abs/2110.10464).
+"""
+struct GeneralizedBuresWassersteinMetric{T<:AbstractMatrix} <: RiemannianMetric
+    M::T
+end
+function GeneralizedBuresWassersteinMetric(M::T) where {T<:AbstractMatrix}
+    return GeneralizedBuresWassersteinMetric{T}(M)
+end
```

Comment:
```suggestion
```
This constructor is not needed.

Reply:
Sure, that was my try to get rid of the 4 new ambiguities, that I have no clue where they come from ;)